This updates avahi-daemon.8 man page to contain information about
svc:/network/dns/multicast:default SMF service and its properties.

Solaris specific, not suitable for upstream.

--- a/man/avahi-daemon.8.xml.in
+++ b/man/avahi-daemon.8.xml.in
@@ -131,6 +131,28 @@
 
 	</options>
 
+	<section name="Service">
+		<p>The<file>avahi-daemon</file>should be managed by the Solaris Management Facility (SMF) and should be administered by the<cmd>svcadm</cmd>(8) command using the following fault management resource identifier (FMRI):</p>
+
+		<p><cmd>svc:/network/dns/multicast:default</cmd></p>
+
+		<p>The configuration options for host name and host address queries using Multicast DNS are stored in a property group named <opt>nss_mdns_config</opt>. The properties that make up the configuration options are as follows:</p>
+
+		<options>
+			<option>
+				<p><opt>search</opt></p>
+				<optdesc><p>A list of domain names for host name look up. By default, no search domains are included in the configuration.</p></optdesc>
+			</option>
+
+			<option>
+				<p><opt>domain</opt></p>
+				<optdesc><p>A list of valid host name and address domains checked before Multicast DNS is used to look up the host name or host address. Host name or address outside domains not listed will not be queried at all. Domains specified in the <opt>search</opt> list are always treated as valid.</p></optdesc>
+			</option>
+		</options>
+
+		<p>Please note the above configuration options are Volatile and may change in a future release.</p>
+			</section>
+
     <section name="Files">
 
       <p><file>@pkgsysconfdir@/avahi-daemon.conf</file>: the default configuration file for avahi-daemon, see <manref name="avahi-daemon.conf" section="5"/> for more information.</p>
